src/mantine-demos/src/Demos.story.tsx

```diff
@@ -100,6 +100,7 @@ attachDemos(storiesOf('Styles', module), demos.CreateStylesDemos);
 attachDemos(storiesOf('Theme', module), demos.ThemeDemos);
 attachDemos(storiesOf('Spotlight', module), demos.SpotlightDemos);
 attachDemos(storiesOf('Form', module), demos.FormDemos);
+attachDemos(storiesOf('NProgress', module), demos.NProgressDemos);
```

Comment:
This file will be removed or already removed, demos should be added other way to avoid performance issues. See Tabs demos for reference.

Reply:
Is that change also on the master branch? Because this is based on the master branch

Reply:
No, this PR is for v5 branch, and it must use it as a base.

Reply:
Okay then I will rebase it later